src/main/java/io/github/jhipster/config/JHipsterProperties.java

```diff
@@ -191,6 +193,8 @@ public Ehcache getEhcache() {
             return ehcache;
         }
 
+        public Infinispan getInfinispan() { return infinispan; }
```

Comment:
It would be better to have this getter this way instead of a one-liner (take `getEhcache` for instance)
```java
public Infinispan getInfinispan() {
  return infinispan;
}
```